website/pages/en/index.js

```diff
@@ -202,12 +174,12 @@ class Index extends React.Component {
 									/>
 								</div>
 								<div className='col-12 col-md-5'>
-									<h2 className='h1'>(Almost) production ready</h2>
+									<h2 className='h1'>Production Ready</h2>
```

Comment:
I'm not convinced of this. We can't consider all of the Frame pallets as production ready.

Reply:
maybe there is a better way to say this then-- (Almost) imo sounds buggy... how about 
'Future ready' or something slightly less literal @danforbes

Reply:
Checked with someone on the comms side and they were okay with this phrasing. Thanks for looking out, @riusricardo :pray: